apps/ui/priv/static/js/widget.js

```diff
@@ -0,0 +1,24 @@
+var iframe = document.createElement("iframe");
+var container = document.getElementById("commentable-thread");
+
+iframe.style.width = `${container.offsetWidth}px`
```

Comment:
@baalberith I think we still intend to have this in assets/ and then gets transpiled to ES5, and then lands in priv in compiled form?

Otherwise we're using here ES6+ syntax

Reply:
Done